qiskit/visualization/__init__.py

```diff
@@ -78,13 +79,23 @@
 `matplotlib.figure.Figure.savefig
 <https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.savefig>`_ method.
 
-.. code-block:: python
+.. plot::
+   :include-source:
+   :nofigs:
+   :context:
 
    plot_histogram(data, filename='new_hist.png')
 
    hist = plot_histogram(data)
    hist.savefig('new_hist.png')
 
+.. plot::
+   :nofigs:
+
+   # This block is hidden from readers. It's cleanup code.
+   from pathlib import Path
+   Path("new_hist.png").unlink()
```

Comment:
Similar thing here - it feels better to me not to have the docs build actually write out to the filesystem as a side effect that we have to tidy up.

Reply:
I agree, I think this one is simple enough that we don't need to test it.